### PR TITLE
Vince/system launch fixes

### PR DESCRIPTION
**Pull request by Dulluhan**
Has launch file changes implemented up to point that system is integrated and running on NUC. Git rebase origin/master doubled the commits.

### CODE REVIEW

File: src/controller/launch/imu.launch [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<!-- Phidgets IMU launch file -->
+
+<launch>
+
+  #### Nodelet manager ######################################################
```
Comment (garethellis0) [inline src/controller/launch/imu.launch]: Just realizing this now, but comments like these should be enclosed with `<!-- -->`, I've had issues before with using invalid comment characters.

File: src/launch/full_system.launch [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    This launch file is responsible for launching the entire entire system
+    (High and Low Level AI/Control, and Serial Communication With Arduino)
+-->
+    <!-- Message Remaps -->
+    <remap from="decision_node/output" to="velocity_publisher"/>
+    <remap from="arduino" to="Arduino"/>
+    <remap from="imu/data" to="imu_data" />
+    <remap from="gateDetect/output" to="gate_location" />"
+
+    <!-- TODO: Start Camera Here -->
```
Comment (garethellis0) [inline src/launch/full_system.launch]: Should remove TODO

**Comment by garethellis0** [comment]
Looks good sans the two comments, I think we should wait until @Dulluhan indicates this is good to go before merging, unless there's a reason we need this in `master` asap.

**Comment by garethellis0** [comment]
Whoops, sorry, closed by accident.

File: src/controller/launch/imu.launch (hunk 1 above)
Comment (newtoncn) [reply src/controller/launch/imu.launch]: This is straight from phidgets website, which makes me think it should be alright

**Pull request merged**